README.tfdoc.hcl

```diff
@@ -1091,56 +1091,100 @@ section {
     title   = "Module Outputs"
     content = <<-END
       The following attributes are exported by the module:
+    END
 
-      - **`repository`**
-
-        All repository attributes as returned by the [`github_repository`] resource containing all arguments as specified above and the other attributes as specified below.
-
-        - **`full_name`**
-
-          A string of the form "orgname/reponame".
-
-        - **`html_url`**
-
-          URL to the repository on the web.
-
-        - **`ssh_clone_url`**
-
-          URL that can be provided to git clone to clone the repository via SSH.
-
-        - **`http_clone_url`**
-
-          URL that can be provided to git clone to clone the repository via HTTPS.
-
-        - **`git_clone_url`**
+    output "repository" {
+      type        = object(repository)
+      description = <<-END
+        All repository attributes as returned by the [`github_repository`]
+        resource containing all arguments as specified above and the other
+        attributes as specified below.
```

Comment:
```suggestion
        All repository attributes as returned by the [`github_repository`] resource containing all arguments as specified above and the other attributes as specified below.
```

Reply:
Hmm. I broke these up into multiple lines for easier readibility, especially here on Github. Had a conversation about this with Tiago [here](https://github.com/mineiros-io/terraform-google-cloud-sql-proxy/pull/7#discussion_r786714166). Let me know what you think @thiesen

Reply:
got it! my only concern is that I would expect the text to be displayed as defined in the input (even though this the rendered document shows everything in the same line) so this brings me to the question: do you believe this is the correct behaviour or should we iterate on it on Terradoc side so the text we input matches what is rendered?